src/libcollections/string.rs

```diff
@@ -1015,8 +1015,7 @@ impl String {
     }
 
     /// Create a draining iterator that removes the specified range in the string
-    /// and yields the removed chars from start to end. The element range is
-    /// removed even if the iterator is not consumed until the end.
```

Comment:
Now I understand. But that is somewhat orthogonal isn't it... not worth mentioning in the summary. The examples I added also demonstrate that fact.